pkg/kubectl/cmd/clusterinfo.go

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2015 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/resource"
+
+	"github.com/spf13/cobra"
+)
+
+func (f *Factory) NewCmdClusterInfo(out io.Writer) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "clusterinfo",
+		Short: "Display cluster info",
+		Long: `Display IPs of the master and the following services running in cluster:
+	Grafana, Elasticsearch and Kibana`,
+		Run: func(cmd *cobra.Command, args []string) {
+			RunClusterInfo(f, out, cmd)
+		},
+	}
+	return cmd
+}
+
+func RunClusterInfo(f *Factory, out io.Writer, cmd *cobra.Command) {
+	client, err := f.ClientConfig(cmd)
+	checkErr(err)
+	fmt.Fprintf(out, "Kubernetes master is running at %v\n", client.Host)
+
+	printPodInfo(f, out, cmd, "name=influxGrafana", "Grafana dashboard", 0)
```

Comment:
Maybe rather than hardcoding these we should define a label convention for "cluster services" and then apply that label.  For extra rationale, you forgot to include the DNS server in this list.

Perhaps we should add a "kubernetes.io/cluster-service" label to each object in these chains (with the value being the name string you want to show up in such a CLI - we can do "label exists" queries, right?)

@bgrant0607